regexes/device/mobiles.yml

```diff
@@ -910,6 +921,12 @@ Asus:
     - regex: 'Slider SL101'
       model: 'Eee Pad Slider SL101'
       device: 'tablet'
+    - regex: 'P008 Build[);/ ]'
+      model: 'ZenPad Z8'
+      device: 'tablet'
+    - regex: 'P027 Build[);/ ]'
```

Comment:
having the `[);/ ]` here is not really needed, as it is already matched in the overall regex and `Pxxx Build` should be enough for an exact match then

Reply:
Indeed, changed!